### DISCUSSION TITLE
Welcome to Nova Discussions!

### DISCUSSION THREAD

**Discussion by srinathsetty** [microsoft/Nova#74] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Gorzorg** [comment]
Hi, I am a programmer who is trying to approach this framework, mainly but not only motivated by blockchain applications.

What the project promises seems very cool, but I am having a hard time understanding how to properly use nova, mainly because of my lack of preparation, but also because of the lack of a comprehensive tutorial.

Is it more appropriate for me to ask my questions here, or is it better if I open a new discussion?
